sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleExponentialHistogramBuckets.java

```diff
@@ -249,6 +251,35 @@ int getScaleReduction(long newStart, long newEnd) {
     return scaleReduction;
   }
 
+  private int getIndexByLogarithm(double value) {
+    return (int) Math.floor(Math.log(value) * scaleFactor);
```

Comment:
I guess `(int)` cast without the floor would behave the same

Reply:
Not for negative values: 
```
(int) -2.2  == -2
(int) Math.floor(-2.2) == -3
```

Reply:
But then again, this function should never receive negative values 🤔